tests/generators/OperationGenerator.ts

```diff
@@ -144,23 +144,60 @@ export default class OperationGenerator {
     const [updatePublicKey] = await Jwk.generateEs256kKeyPair();
 
     const delta = {
-      update_commitment: Multihash.canonicalizeThenDoubleHashThenEncode(updatePublicKey),
+      updateCommitment: Multihash.canonicalizeThenDoubleHashThenEncode(updatePublicKey),
       patches
     };
 
     const deltaHash = Multihash.canonicalizeThenHashThenEncode(delta);
 
     const suffixData = {
-      delta_hash: deltaHash,
-      recovery_commitment: Multihash.canonicalizeThenDoubleHashThenEncode(recoveryPublicKey)
+      deltaHash: deltaHash,
+      recoveryCommitment: Multihash.canonicalizeThenDoubleHashThenEncode(recoveryPublicKey)
     };
 
     const didUniqueSuffix = CreateOperation['computeJcsDidUniqueSuffix'](suffixData);
 
     const shortFormDid = network ? `did:sidetree:${network}:${didUniqueSuffix}` : `did:sidetree:${didUniqueSuffix}`;
 
     const initialState = {
-      suffix_data: suffixData,
+      suffixData: suffixData,
+      delta: delta
+    };
+
+    const canonicalizedInitialStateBuffer = JsonCanonicalizer.canonicalizeAsBuffer(initialState);
+    const encodedCanonicalizedInitialStateString = Encoder.encode(canonicalizedInitialStateBuffer);
+
+    const longFormDid = `${shortFormDid}:${encodedCanonicalizedInitialStateString}`;
+    return {
+      longFormDid,
+      shortFormDid,
+      didUniqueSuffix
+    };
+  }
+
+  /**
+   * Generates a long from from create operation data.
+   */
+  public static async longFormFromCreateOperationData (createOperationData:any, network?: string) {
```

Comment:
Nit: Should have space between : and any

Reply:
eslint would've picked this up, but too bad since the switch from tslint to eslint a few days ago, we still have ~200 lint issues to resolve.

Reply:
I would prefer not to resolve lint fixes in this PR, it's already big enough.

Reply:
Agree, as long as we don't increase the total eslint erro count.

Reply:
You can cause PRs to fail if lint rules fail using CI, and you can also setup husky to prevent commits where lint errors occur. I am going to resolve this, as that work should happen in a subsequent PR.

Reply:
let's add the space here because it's a change that would create a new error, and we'll go back and fix the subsequent errors in a later PR (as well as add the CI changes to make linting fail the pipeline)

Reply:
Fixed.